components/cards/SessionCard.tsx

```diff
@@ -0,0 +1,275 @@
+ * @param screen: 'home' | 'sessions', default is 'home'
+ * @param variant: 'card' | 'list'
+ * @returns SessionCard Component
+ */
+
+const SessionCardOnHome = (props: Omit<SessionCardHome, 'screen'>) => {
+  const { handlePress, item } = props;
+  const { colors } = useTheme();
+
+  return (
+    <TouchableWithoutFeedback testID="card" onPress={handlePress} disabled={item.slug.length < 1}>
+      <View style={[styles.card, { backgroundColor: colors.card }]}>
+        <Image
+          source={{ uri: item.session_image || '' }}
+          style={styles.image}
+          contentFit="cover"
+          contentPosition="left"
+        />
+        <Space size={8} />
+        <View style={styles.bottom}>
+          <View style={styles.description}>
+            <StyledText font="bold" numberOfLines={2} style={styles.title}>
+              {truncate(50, item.title)}
+            </StyledText>
+          </View>
+
+          <Space size={12} />
+
+          <StyledText size="sm" font="light">
+            {getSessionTimeAndLocation(item.slug)}
+          </StyledText>
+        </View>
+      </View>
+    </TouchableWithoutFeedback>
+  );
+};
+
+const SessionCardOnSessions = (props: Omit<SessionCardSessions, 'screen'>) => {
+  const { handlePress, item, handleBookMark } = props;
+  const { colors } = useTheme();
+  const router = useRouter();
+  return (
+    <TouchableWithoutFeedback testID="card-sessions" onPress={handlePress} disabled={item.slug.length < 1}>
+      <View style={[styles.card, styles.cardTertiary, { backgroundColor: colors.card }]}>
+        {item.session_image !== null ? (
+          <Image
+            source={{
+              uri:
+                item.session_image ||
+                'https://res.cloudinary.com/droidconke/image/upload/v1668083311/prod/upload/sessions/pwt9pnojtmng8bymxkls.png',
+            }}
+            style={styles.image}
+            contentFit="cover"
+            contentPosition="left"
+          />
+        ) : (
+          <Image
+            source={require('../../assets/images/banner.png')}
+            style={styles.image}
+            contentFit="cover"
+            contentPosition="center"
+          />
+        )}
+
+        <Space size={4} />
+
+        <View style={styles.bottomForSessions}>
+          <StyledText size="sm" font="light">
+            {getScheduleTimeAndLocation(item.start_date_time, item.end_date_time, item.rooms[0])}
+          </StyledText>
+          <Space size={11} />
+          <View>
+            <StyledText font="bold" numberOfLines={2} style={styles.title}>
+              {truncate(50, item.title)}
+            </StyledText>
+          </View>
+          <Space size={16} />
+          <Row>
+            <Row style={styles.avatarRow}>
+              {item.speakers.map((speaker) => (
+                <Pressable
+                  key={speaker.avatar}
+                  onPress={() => router.push({ pathname: '../speaker', params: speaker })}
```

Comment:
Clicking on the avatar throws this error:

```
Error: Unable to open URL: ../speaker?name=Benard%20Ngoda&tagline=@ibensalcie&biography=I%20am%20a%20student%20at%20Moi%20University%20doing%20indu ... ally.com
```

Reply:
Uhm, did you get this error on mobile or on web?

Reply:
On mobile.

Reply:
Ok, lemme look into it.

Reply:
the comment on my recent review should fix this error. just implement it as in the comment

Reply:
Ok, done!